src/components/DataTable/index.jsx

```diff
@@ -115,9 +145,36 @@ export default class DataTable extends Component {
               </TableCell>
             </TableRow>
           ) : (
-            items.map(renderRow)
+            items
+              .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+              .map(renderRow)
           )}
         </TableBody>
+        {isPaginate && (
+          <TableFooter>
+            <TableRow>
+              <TablePagination
```

Comment:
![Screen Shot 2019-03-13 at 9 03 49 AM](https://user-images.githubusercontent.com/3766511/54280913-f4f61d80-456e-11e9-9348-87c94eed308a.png)

We should:
* Have the TablePagination outside the table. This would make sure the pagination is always displayed to the user even when the table is being viewed on a small screen where scrolling is required to view the data. There's an example of that in https://material-ui.com/demos/tables/#sorting-amp-selecting. Basically you will need to remove `TableFooter` and `TableRow` and then have `TablePagination` outside the table.
* The table should be scrollable similar to how all the tables are in the material site.